lang/chicken-scheme/patches/00.build

```diff
@@ -0,0 +1,15 @@
+diff -Nur chicken-5.4.0/defaults.make chicken-5.4.0-new/defaults.make
```

Comment:
1st) Have you check how other patches in this repository looks like? 
That would provide you answers that you are looking for.
2nd) Why do we need to carry this patch in downstream?

Reply:
Ah, ok.
1) Didn't think the first line would make a difference; I've removed it.
2) I don't understand the question. Do you mean I should include an explanation of why it's needed, in the PR? I will do that.

Reply:
> Didn't think the first line would make a difference; I've removed it.

Indeed, it doesn't make a difference when compiling the package... It only shows up when the test bot looks at it (?).

> I will do that.

Done!

Reply:
I havent said anything about the first line.

Reply:
> I havent said anything about the first line.

I don't get it. I have run, locally,

```
make package/chicken-scheme/clean
make package/chicken-scheme/refresh
```

and it all works fine (no errors, returns status zero). But when the PR goes through the tests, it complains about dirty patches. 

It seems that here (my notebook), in the second time it goes through the patch, all is fine:

```
Applying patch 00.build
patching file defaults.make

Now at patch 00.build
touch "/home/jeronimo/pkg/openwrt/build_dir/target-mips_24kc_musl/chicken-5.4.0/.quilt_checked"
Applying patch 00.build
patching file defaults.make

Now at patch 00.build
Patch 00.build is unchanged
rm -rf /home/jeronimo/pkg/openwrt/package/chicken-scheme/patches 2>/dev/null >/dev/null
```

but during the automated tests,

```
  Applying patch 00.build
  patching file defaults.make
  
  Now at patch 00.build
  touch "/builder/build_dir/target-aarch64_generic_musl/chicken-5.4.0/.quilt_checked"
  Applying patch 00.build
  patching file defaults.make
  
  Now at patch 00.build
  Refreshed patch 00.build
  rm -rf /feed/lang/chicken-scheme/patches 2>/dev/null >/dev/null
```

Why on my notebook it says "`Patch 00.build is unchanged`" but on github it says "`Refreshed patch 00.build`"?